### PR TITLE
[WIP] Add in FFaker and get a seeding pattern established

### LINKED ISSUE #8: Seed dev databases
Let's have a way for us to populate our development databases with lots of data so we have meaningful data to query.

### PR DESCRIPTION
**Pull request by DerekStride**
I've got an initial pattern up for seeding just for comments, I can think of a few more approaches that we could take.

We might need to revisit this when we start creating problems with our data that we want to replicate.

**Alternatives:**
- Load data from a fixture-like data source (YAML, CSV)
    - Gives us control over values and relationships
    - Can see the relationships in the data files
    - Hard to add data

Fixes https://github.com/GraphQL-Query-Planner/webscale/issues/8

### CODE REVIEW

**Comment by ismail-syed** [comment]
🎩 went well. Had to `git rebase origin/add-migrations` to pick up the work from Justin branch to get this to work.

File: lib/faker.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Faker
+  attr_reader :default_count
+  def initialize(default_count = 5)
```
Comment (ismail-syed) [inline lib/faker.rb:3]: Not a concern atm, but we'll be increasing this as the project grows, right? To a couple hundred possibly.

**Review by ismail-syed — APPROVED** [review]
💯

**Comment by ismail-syed** [comment]
I prefer the approach in your PR and iterating on it. 

> Gives us control over values and relationships

If our ids are incremental, I don't think this will be too difficult to script. 

> Can see the relationships in the data files

Won't it be easier to see the relationships in Sequel Pro, instead of verbose data files

File: lib/faker.rb (hunk 1 above)
Comment (DerekStride) [reply lib/faker.rb:3]: Where this default lives in the code will likely change as we iterate and build relationships with content. The idea is we can set this from the commands we run or set them to defaults we want.

e.g. `rake seed:users NUM=5000`

**Review by justinkrol — APPROVED** [review]
LGTM. I definitely think this is better than fixtures, since we don't really care about the content of the data, and this is much more scalable, as long as we can properly fake associations.

**Pull request merged**